src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -552,17 +555,23 @@ void GcsServer::InitFunctionManager() {
   function_manager_ = std::make_unique<GcsFunctionManager>(kv_manager_->GetInstance());
 }
 
+void GcsServer::InitUsageStatsClient() {
+  usage_stats_client_ = std::make_unique<UsageStatsClient>(
+      "127.0.0.1:" + std::to_string(GetPort()), main_service_);
```

Comment:
how much overhead do you expect from UsageStatsClient?

Reply:
It just makes a gcs kv call so shouldn't be much.